pkgs/by-name/vd/vdhcoapp/ffmpeg-filepicker.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/src/converter.js b/src/converter.js
+index af7b4c3..20da407 100644
+--- a/src/converter.js
++++ b/src/converter.js
+@@ -1,4 +1,4 @@
+-import open from 'open';
++function open() {}
```

Comment:
what's the motivation for making open a noop?

Reply:
as the import statement could not be used this was simply a quick fix, forgotten to be removed, it is now replaced with `const open = import('open')`